docker/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+tracker:
+  image: ${HERON_IMAGE}
+  command: heron-tracker
+  ports:
+    - "8888:8888"
+  volumes:
+    - "herondata:/root/.herondata"
+ui:
+  image: ${HERON_IMAGE}
+  command: heron-ui --tracker_url=http://192.168.33.11:8888
```

Comment:
This ip address will likely not work for anyone else. Is there a better way to link these two containers together without specifying a static ip address?